src/components/SwapForm/OutputCurrencyPanel.tsx

```diff
@@ -76,12 +79,17 @@ const OutputCurrencyPanel: React.FC<Props> = ({
             text={
               <Text fontSize={12}>
                 <Trans>
-                  This is the estimated output amount. Do review the actual output amount in the confirmation stage.
+                  This is the estimated output amount. It is inclusive of any applicable swap fees. Do review the actual
```

Comment:
Tooltip should check CHAINS_SUPPORT_FEE_CONFIGS too.

Reply:
Done